src/libAtomVM/nifs.c

```diff
@@ -31,6 +31,8 @@
 #include <stdio.h>
 #include <string.h>
 #include <time.h>
+#include <unistd.h>
```

Comment:
As I mention later, I'm worried we are leaking unistd and POSIX stuff into platform agnostic nifs.c.